Doc/using/configure.rst

```diff
@@ -297,6 +297,16 @@ General Options
 
    .. versionadded:: 3.13
 
+.. option:: --enable-experimental-jit
+   Enables :ref:`JIT compiler <whatsnew313-jit-compiler>`.
+   Possible values:
+      * ``no`` - build the interpreter without the JIT.
+      * ``yes`` - build the interpreter with the JIT.
+      * ``yes-off`` - build the interpreter with the JIT but disable it by default.
+      * ``interpreter`` - build the interpreter without the JIT, but with the tier 2 enabled interpreter.
+
+   .. versionadded:: 3.13
```

Comment:
```suggestion
.. option:: --enable-experimental-jit=[no|yes|yes-off|interpreter]

   Indicate how to integrate the :ref:`JIT compiler <whatsnew313-jit-compiler>`.

   * ``no`` - build the interpreter without the JIT.
   * ``yes`` - build the interpreter with the JIT.
   * ``yes-off`` - build the interpreter with the JIT but disable it by default.
   * ``interpreter`` - build the interpreter without the JIT, but with the tier 2 enabled interpreter.

   .. versionadded:: 3.13
```

Reply:
Is there a default value if you just pass `--enable-experimental-jit`?

Reply:
> Is there a default value if you just pass `--enable-experimental-jit`?

Yes, it's a `yes`

Reply:
I think we should mention it then. Maybe by adding some "(default)" after the `yes` (either before the `-` or at the end of the option's description, up to you). Or even after the bullet list: 

```rst
By default, ``--enable-experimental-jit`` implies ``--enable-experimental-jit=yes``.
```